### ISSUE TITLE
Doesn't work with Firefox 29

### ISSUE THREAD

**Issue by legoscia** [flattr/fx-flattr-addon#2]
When using this addon with Firefox 29 (currently in the Aurora branch), the Flattr icon doesn't appear in the URL bar on e.g. Github commit pages.

It seems like this happens because showforpage requires `sdk/deprecated/tab-browser.js`, but this file was removed (see [Bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=847593) and [Github](https://github.com/mozilla/addon-sdk/pull/1328)).

**Comment by voxpelli** [comment]
I've now updated the "showforpage" module to no longer have a dependency on "tab-browser" and I have verified that the same pages that at first didn't work in the latest Firefox Aurora version, 29.0a2 (2014-02-26), now work when the Flattr add-on is built with the new 0.2.2 version of "showforpage".

There still of course needs to be a new build of the Flattr add-on submitted to the addons.mozilla.org page, but apart from that this can be considered fixed now. Thanks for pointing it out!
